parse offset-list for disc element

JonnyJD opened the pull request:
see [MBS-8180](http://tickets.musicbrainz.org/browse/MBS-8180)

XML looks like this:
http://musicbrainz.org/ws/2/discid/xp5tz6rE4OHrBafj0bLfDRMGK48-
```
<offset-list count="8">
  <offset position="1">182</offset>
  <offset position="2">33322</offset>
  <offset position="3">52597</offset>
  <offset position="4">73510</offset>
  <offset position="5">98882</offset>
  <offset position="6">136180</offset>
  <offset position="7">169185</offset>
  <offset position="8">187490</offset>
</offset-list>
```

They are returned by our API as a list: `[182, 33322, 52597, 73510, 98882, 136180, 169185, 187490]` which is exactly what the json WS is also doing.
The number of tracks on the disc can now be extracted using `result['disc']['offset-count']`.

The new data is already used in the [find_disc.py example](https://github.com/JonnyJD/python-musicbrainzngs/blob/11bb79085cba7a9ffc4f7653d6c0fc285c4ccc8f/examples/find_disc.py#L34).

alastair commented on musicbrainzngs/mbxml.py:
```diff
@@ -505,7 +505,9 @@ def parse_disc(disc):
     elements = ["sectors"]
-    inner_els = {"release-list": parse_release_list}
+    inner_els = {"release-list": parse_release_list,
+                 "offset-list": parse_offset_list
```
Your docs are now out of date
https://github.com/JonnyJD/python-musicbrainzngs/blob/offset-list/musicbrainzngs/musicbrainz.py#L969

JonnyJD replied:
Thanks, I forgot that I manually added info about the returned data. (fixed with next commit)

alastair commented on musicbrainzngs/mbxml.py:
```diff
@@ -523,6 +525,9 @@ def parse_cdstub(cdstub):
 
     return result
 
+def parse_offset_list(ol):
```
Can we have an updated test (or a test for this method if it doesn't already exist)

JonnyJD replied:
The only test for `get_releases_by_discid` tests the created WS url, not the result. (`test_getentity`).

I'll see if I can create a `test_mbxml_disc.py` or similar.

alastair replied:
cool - it'd be nice to see xml tests for each method and some combination of their includes

JonnyJD replied:
tests added in [test_mbxml_discid.py](https://github.com/JonnyJD/python-musicbrainzngs/blob/35222a08be8c8a817fdc86bd2c47c2e1c09063df/test/test_mbxml_discid.py)

JonnyJD commented:
I added mbxml tests, which runs fine as of:
https://travis-ci.org/JonnyJD/python-musicbrainzngs/builds/66026808
(my fork is automatically built by travis, but this repo isn't so the test status is not displayed in the PR).

I also added a note in `CHANGES`.

alastair commented:
:ship: it
